tests/e2e/tidbcluster/stability-br.go

```diff
@@ -260,15 +262,15 @@ func testBR(provider, ns string, fw portforward.PortForward, c clientset.Interfa
 		backup.Spec.From.TLSClientSecretName = &backupSecretName
 	}
 	_, err = cli.PingcapV1alpha1().Backups(ns).Create(backup)
-	framework.ExpectNoError(err)
+	framework.ExpectNoError(err, "failed to create backup in ns %s: %v", ns, backup)
 
 	cleanFunc := func() {
 		// delete backup data in S3
 		err = cli.PingcapV1alpha1().Backups(ns).Delete(backup.Name, &metav1.DeleteOptions{})
-		framework.ExpectNoError(err)
+		framework.ExpectNoError(err, "failed to delete backup in ns %s, ")
```

Comment:
missing `ns` for `%s`, and maybe you can include `backup.Name` too (or include it for bellow lines?).